gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -1,286 +0,0 @@
-"""
```

Comment:
we have removed this test while management command for this test is still there.